### ISSUE TITLE
function get_exact_association nearly the same as get_property

### ISSUE THREAD

**Issue by Cudok** [BIM2SIM/bim2sim#617]
the function get_exact_association is nearly the same as get_property.

both functions are part of base_elements.py (see line 486 ff).

Maybe it is not right.

@DaJansenGit  could you check it?

**Comment by DaJansenGit** [comment]
you are right, that was indeed a doubled function. I removed the function, changed the code and created a PR.
